pkg/kubelet/container_bridge.go

```diff
@@ -82,6 +82,16 @@ func ensureCbr0(wantCIDR *net.IPNet) error {
 		}
 		return createCBR0(wantCIDR)
 	}
+	// Put the container bridge into promiscuous mode to force it to accept hairpin packets.
```

Comment:
I want to set it every time we check networking (every 30s) after the bridge is created. Your point about not doing it the first time is valid, though, so now I'm just falling-through.